frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -36,6 +36,224 @@ const connector = connect(
+  min-width: 309px;
+  border-radius: 0px 12px 0px 0px;
+
+  padding: 64px 64px 64px 64px;
+
+  background: ${CLOUD};
+
+  @media (max-width: 600px) {
+    width: 100%;
+    min-height: 347px;
+    border-radius: 0px 0px 0px 0px;
+    padding: 48px 24px 24px 24px;
+  }
+`;
+
+// HELP
+// Thinking of making a 2-column grid with the left-hand column being fixed-width for icons
+const MenuButtonContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 16px;
+
+  background: red;
+  width: 100%;
+  height: 100px;
+`;
+
+const TitleText = styled.p`
+  font-family: "Rubik";
+  font-size: 26px;
+  font-weight: 700;
+  text-transform: lowercase;
+  word-wrap: break-word;
+
+  @media (max-width: 600px) {
+    font-size: 30px;
+  }
+`;
+
+const ProfilePicture = styled.img`
+  margin-bottom: 24px;
+
+  height: 183px;
+  width: 183px;
+
+  @media (max-width: 600px) {
+    font-size: 30px;
+    height: 148px;
+    width: 148px;
+  }
+`;
+
+const SessionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+
+  width: 48%;
+  min-height: 100%;
+  padding-left: 64px;
+  padding-top: 64px;
+  padding-bottom: 64px;
+
+  @media (max-width: 600px) {
+    min-width: 100%;
+    padding-top: 48px;
+    padding-right: 24px;
+    padding-left: 24px;
+  }
+`;
+
+const NextSessionButton = styled(PurpleButton)`
+  min-width: 32%;
+  height: 100px;
+  margin-bottom: 64px;
+  text-transform: lowercase;
+
+  @media (max-width: 600px) {
+    min-width: 100%;
+  }
+`;
+
+const WeekProgressContainer = styled.div`
+  width: 36%;
+  min-height: 100%;
+
+  padding-left: 64px;
+  padding-right: 64px;
+  padding-top: 64px;
+
+  @media (max-width: 600px) {
+    min-width: 100%;
+    padding-top: 48px;
+    padding-right: 24px;
+    padding-left: 24px;
+  }
+`;
+
+const ProgressStatsContainer = styled.div`
+  display: flex;
+  flex: 1;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  min-width: 100%;
+`;
+
+const ProgressBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 30px 30px 30px 30px;
+
+  flex: none;
+  order: 0;
+  flex-grow: 1;
+
+  max-width: calc(50% - 16px);
+  margin-bottom: 32px;
+
+  background: ${CLOUD};
+  border-radius: 12px;
+
+  @media (max-width: 600px) {
+    max-width: calc(50% - 12px);
+    margin-bottom: 24px;
+  }
+`;
+
+const ProgressStatNumber = styled.p`
+  font-family: "Rubik";
+  font-size: 56px;
+  font-weight: 700;
+  text-transform: lowercase;
+  color: ${INK};
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0px 0px;
+
+  @media (max-width: 600px) {
+    font-size: 36px;
+  }
+`;
+
+const ProgressStatDescription = styled.p`
+  text-transform: lowercase;
+  font-family: Open Sans;
+  font-size: 18px;
+  font-weight: 400;
+  text-transform: lowercase;
+  word-wrap: break-word;
+  text-align: center;
+
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 0px 0px;
+
+  @media (max-width: 600px) {
```

Comment:
Based on the largeness of the content, we should probably go to the mobile view at 900px, not 600 in this circumstance. The small view looks kinda weird now.